graphene_mongo/converter.py

```diff
@@ -109,11 +109,23 @@ def convert_field_to_list(field, registry=None):
             return
         base_type = base_type._type
 
+    if (
+        isinstance(base_type, (graphene.Field))
+        and hasattr(base_type._type, "_meta")
+        and isinstance(base_type._type._meta, graphene.types.union.UnionOptions)
+    ):
+        base_type = base_type._type
+
```

Comment:
Not sure if this is the most elegant way of checking if the type is a Union but since this is a synthetic and dynamic type, I couldn't think of a different way to do it.

Reply:
@abawchen